nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/admin/SetRequestTracingConfiguration.java

```diff
@@ -166,63 +167,72 @@ public void execute(AdminCommandContext context) {
+                            warn = !traceStoreSize.toString().equals(proxy.getTraceStoreSize());
+                            proxy.setTraceStoreSize(traceStoreSize.toString());
                         }
                         if (traceStoreTimeout != null) {
-                            requestTracingServiceConfigurationProxy.setTraceStoreTimeout(traceStoreTimeout);
+                            warn = !traceStoreTimeout.equals(proxy.getTraceStoreTimeout());
+                            proxy.setTraceStoreTimeout(traceStoreTimeout);
                         }
                         if (reservoirSamplingEnabled != null) {
-                            requestTracingServiceConfigurationProxy.setReservoirSamplingEnabled(reservoirSamplingEnabled.toString());
+                            proxy.setReservoirSamplingEnabled(reservoirSamplingEnabled.toString());
                         }
                         
                         if (historicTraceStoreEnabled != null) {
-                            requestTracingServiceConfigurationProxy.setHistoricTraceStoreEnabled(historicTraceStoreEnabled.toString());
+                            proxy.setHistoricTraceStoreEnabled(historicTraceStoreEnabled.toString());
                         }
                         if (historicTraceStoreSize != null) {
-                            requestTracingServiceConfigurationProxy.setHistoricTraceStoreSize(historicTraceStoreSize.toString());
+                            warn = !historicTraceStoreSize.toString().equals(proxy.getHistoricTraceStoreSize());
+                            proxy.setHistoricTraceStoreSize(historicTraceStoreSize.toString());
                         }
                         if (historicTraceStoreTimeout != null) {
-                            requestTracingServiceConfigurationProxy.setHistoricTraceStoreTimeout(historicTraceStoreTimeout);
+                            warn = !historicTraceStoreTimeout.equals(proxy.getHistoricTraceStoreTimeout());
+                            proxy.setHistoricTraceStoreTimeout(historicTraceStoreTimeout);
                         }
-
-                        actionReport.setActionExitCode(ActionReport.ExitCode.SUCCESS);
-                        return requestTracingServiceConfigurationProxy;
+                        actionReport.setActionExitCode(ExitCode.SUCCESS);
+                        if (warn) {
+                            actionReport.setMessage(strings.getLocalString(
+                                    "requesttracing.configure.store.size.warning",
+                                    "Please note that from 5.194 onwards the store size refers to the size of a single store shared by all cluster members. Any configuration change regarding the store should be applied to all configurations equally to prevent unbalanced sharing. All affected configurations will be extracted and moved to a central configuration in a future release."));
```

Comment:
Why default value needs to be passed if already exists in default `LocalStrings.properties`.

Reply:
Are you saying it can be left out? I just followed what I found was done for other usages of the method.

Reply:
Yes, duplicating text value can be removed, e.g:
````
import com.sun.enterprise.universal.i18n.LocalStringsImpl;
LocalStringsImpl STRINGS = new LocalStringsImpl(SetRequestTracingConfiguration.class);
actionReport.setMessage(STRINGS.get("requesttracing.configure.store.size.warning"));
````

Reply:
Nice, let me try that.